pkg/tecdsa/signing/message.go

```diff
@@ -150,4 +150,24 @@ func (trsm *tssRoundSixMessage) SenderID() group.MemberIndex {
 // marshaling purposes.
 func (trfm *tssRoundSixMessage) Type() string {
 	return messageTypePrefix + "tss_round_six_message"
+}
+
+// tssRoundSevenMessage is a message payload that carries the sender's
+// TSS round seven components.
+type tssRoundSevenMessage struct {
+	senderID group.MemberIndex
+
+	payload   []byte
+	sessionID string
+}
+
+// SenderID returns protocol-level identifier of the message sender.
+func (trsm *tssRoundSevenMessage) SenderID() group.MemberIndex {
+	return trsm.senderID
+}
```

Comment:
I am not sure if this function is needed, see [here](https://github.com/keep-network/keep-core/pull/3243#discussion_r964665190) please.

Reply:
Same response as in https://github.com/keep-network/keep-core/pull/3243#discussion_r964665190